reporting/api/src/main/java/org/jboss/windup/reporting/model/JarDependencyReportToProjectEdgeModel.java

```diff
@@ -0,0 +1,41 @@
+package org.jboss.windup.reporting.model;
+
+import org.jboss.windup.graph.model.ArchiveModel;
+import org.jboss.windup.graph.model.ProjectModel;
+import org.jboss.windup.graph.model.WindupEdgeFrame;
+
+import com.tinkerpop.frames.InVertex;
+import com.tinkerpop.frames.OutVertex;
+import com.tinkerpop.frames.Property;
+import com.tinkerpop.frames.modules.typedgraph.TypeValue;
+
+
+/**
+ * @author mnovotny
+ *
+ */
+@TypeValue(JarDependencyReportToProjectEdgeModel.TYPE)
+public interface JarDependencyReportToProjectEdgeModel extends WindupEdgeFrame
```

Comment:
if that is related to the comment https://github.com/windup/windup/pull/966#discussion_r66870655 I understand it like it should be connected with DependencyModel, right?